core/data.go

```diff
@@ -606,7 +607,7 @@ func ConvertToPaymentMetadata(ph *commonpb.PaymentHeader) *PaymentMetadata {
 
 // OperatorInfo contains information about an operator which is stored on the blockchain state,
 // corresponding to a particular quorum
-type ActiveReservation struct {
+type ReservedPayment struct {
 	SymbolsPerSecond uint64 // reserve number of symbols per second
 	//TODO: we are not using start and end timestamp, add check or remove
```

Comment:
Is this TODO still relevant? Maybe we can update the documentation here for two timestamps.

Also nit: comments should be line above the fields starting with the name of the field